### PR TITLE
DELEGATION: Rename delegations contract to to delegation

### PR DESCRIPTION
**Pull request by vistra**

### CODE REVIEW

File: test/driver.spec.ts [hunk 1]
```diff
@@ -57,7 +57,7 @@ describe('testkit', async () => {
       orbsTokenAddress,
       bootstrapTokenAddress,
       contractRegistryAddress,
-      delegationsAddress,
+      delegationAddress: delegationsAddress,
```
Comment (OdedWx) [inline test/driver.spec.ts:60]: why delegationsAddress?

File: test/driver.spec.ts [hunk 2]
```diff
@@ -38,7 +38,7 @@ describe('testkit', async () => {
     const bootstrapTokenAddress =  dd.bootstrapToken.address;
 
     const contractRegistryAddress = dd.contractRegistry.address;
-    const delegationsAddress = dd.delegations.address;
+    const delegationsAddress = dd.delegation.address;
```
Comment (OdedWx) [inline test/driver.spec.ts:41]: delegationAddress?

Comment (vistra) [reply test/driver.spec.ts:41]: Right

File: test/driver.spec.ts (hunk 1 above)
Comment (vistra) [reply test/driver.spec.ts:60]: Right

**Review by OdedWx — APPROVED** [review]

**Pull request closed without merging**